lib/bolt/pal/yaml_plan/step/verbose.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Bolt
+  class PAL
+    class YamlPlan
+      class Step
+        class Verbose < Step
+          def self.allowed_keys
+            super + Set['verbose']
+          end
+
+          def self.required_keys
+            Set['verbose']
+          end
+
+          # Returns an array of arguments to pass to the step's function call
+          #
+          private def format_args(body)
+            [body['verbose']]
+          end
+
+          # Returns the function corresponding to the step
+          #
+          private def function
+            'out::verbose'
+          end
+
+          # Transpiles the step into the plan language
+          #
+          def transpile
+            code = String.new("  ")
+            code << function_call(function, format_args(body))
+            code << "\n"
+          end
```

Comment:
It looks like this actually results in a small bug when transpiling plans, and is present in the `message` step as well. 

This YAML plan:

```yaml
steps:
  - name: verbose
    verbose: verbosity

return: $verbose
```

Becomes this Puppet plan:

```puppet
plan bolt::yaml() {
  out::verbose('verbosity')

  return $verbose
}
```

`name` keys in YAML plans are equivalent to variable declarations in Puppet. However, the transpile method for `message` and `verbose` isn't declaring these variables, so the above Puppet plan will actually error. We should remove the `transpile` method override in `message` and `verbose`, since the base class' method will do what we want.

Reply:
Would we need any additional testing to make sure that this works, or would the existing testing (plus checking the conversion) be sufficient? Where would I check to see how to manually test the conversion.

Reply:
I don't think an extra test is needed here, just a manual check should be fine.